scripts/export_uml_json.py

```diff
@@ -0,0 +1,146 @@
+"""
+
+Convert a umls release to a json file of concepts.
+
+"""
+import sys
+import json
+import plac
+
+# preferred definitions (from S2)
+DEF_SOURCES_PREFERRED = {'NCI_BRIDG', 'NCI_NCI-GLOSS', 'NCI', 'GO', 'MSH', 'NCI_FDA'}
+
+
+@plac.annotations(
+        output_filename=('name of the output json file', 'positional', None, str),
+        umls_meta_directory=('path of the META directory', 'positional', None, str))
+def main(output_filename, umls_meta_directory):
+
+    concepts_filename = '{}/MRCONSO.RRF'.format(umls_meta_directory)
+    types_filename = '{}/MRSTY.RRF'.format(umls_meta_directory)
+    definitions_filename = '{}/MRDEF.RRF'.format(umls_meta_directory)
+    file_descriptors = '{}/MRFILES.RRF'.format(umls_meta_directory)  # to get column names
+
+    concepts_header = types_header = definitions_header = None
+    with open(file_descriptors) as fin:
+        for line in fin:
+            splits = line.split('|')
+            filename = splits[0]
+            column_names = (splits[2] + ',').split(',')  # ugly hack because all files end with an empty column
+            if filename == 'MRCONSO.RRF':
+                concepts_header = column_names
+                print('Headers for MRCONSO.RRF: {}'.format(concepts_header))
+            elif filename == 'MRSTY.RRF':
+                types_header = column_names
+                print('Headers for MRSTY.RRF: {}'.format(types_header))
+            elif filename == 'MRDEF.RRF':
+                definitions_header = column_names
+                print('Headers for MRDEF.RRF: {}'.format(definitions_header))
+
+    if concepts_header is None or  types_header is None or definitions_header is None:
+        print('ERROR .. column names are not found')
+        exit()
+
+    concept_details = {}  # dictionary of concept_id -> concept_dictionary
```

Comment:
What is a concept dictionary?

I would suggest using a `NamedTuple` here, e.g:

from typing import NamedTuple
```python
class ConceptInformation(NamedTuple):
	concept_id: str
	aliases: List[str]
	types: List[str]
```